.travis.yml

```diff
@@ -2,15 +2,24 @@
 language: python
 sudo: false
 
-env:
-  - TOXENV=py26
-  - TOXENV=py27
-  - TOXENV=py32
-  - TOXENV=py33
-  - TOXENV=py34
-  - TOXENV=pypy
-  - TOXENV=pypy3
-  - TOXENV=cover
+matrix:
+    include:
+        - python: 2.7
+          env: TOXENV=py27
+        - python: 3.4
+          env: TOXENV=py34
+        - python: 3.5
+          env: TOXENV=py35
+        - python: 3.6
+          env: TOXENV=py36
+        - python: nightly
+          env: TOXENV=py37
```

Comment:
I believe that `nightly` should actually be a 3.8 snapshot.  Can you look at what e.g. [`zope.interface` is doing to support 3.7](https://github.com/zopefoundation/zope.interface/blob/e6693826ab7e8c3b640763f65da2f28fe89a961a/.travis.yml#L13).  I realize that WebOb isn't yet doing that.